vspace/src/main/webapp/WEB-INF/views/exhibition/downloads/downloadList.html

```diff
@@ -0,0 +1,214 @@
+<!DOCTYPE html>
+<html layout:decorate="~{layouts/main_staff}">
+<head>
+<script th:src="@{/resources/bootpag/js/bootpag.min.js}">
+</script>
+<script th:inline="javascript">
+    $(document).ready(function($) {
+    	$("#downloads").addClass("active");	
+    	$('.downloadPage-selection').bootpag({
+    	total: [[${downloadsTotalPages}]],
+    	page: [[${downloadsCurrentPageNumber}]],
+    	maxVisible: 10,
+    	leaps: true,
+    	firstLastUse: true,
+    	first: '|<',
+    	last: '>|',
+    	wrapClass: 'pagination',
+    	activeClass: 'active',
+    	disabledClass: 'disabled',
+    	nextClass: 'next',
+    	prevClass: 'prev',
+    	lastClass: 'last',
+    	firstClass: 'first'
+    }).on("page", function(event, num){
+    		window.location.assign("[(@{'/staff/downloads/list'})]"+'?'+"downloadsPagenum="+num);
+    });
+    checkSnapshotCreationStatus('');
+})
+
+    function checkSnapshotCreationStatus(id){
+    	$.ajax({
+            type : "GET",
+            url : "[(@{'/staff/exhibit/task/snapshot/status?'+ ${_csrf.parameterName}+'='+${_csrf.token} })]"+"&snapshotTaskId="+id,
+            cache : false,
+            
+            contentType : false,
+            processData : false,
+            
+            success : function(data) {
+                if(data.taskComplete ==  false) {
+                    pollForExhibitionDownload(data.exhibitionSnapshot.id, data.exhibitionSnapshot.folderName);
+                }
+            },
+            error : function(data) {
+                var loader = document.getElementById('downloadLoader');
+                loader.style.visibility="hidden";
+                
+                var alert = $('<div class="alert alert-danger alert-dismissible fade show" role="alert"><p>Snapshot could not be downloaded</p><button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button></div>');
```

Comment:
Isn't this polling for the status? why does the message say "could not download"?